### DISCUSSION TITLE
Uclibc support?

### DISCUSSION THREAD

**Discussion by KevinRiordan** [pyinstaller/pyinstaller#6347] [Features & Development]
does pyinstaller work with uclibc? or does it require glibc?

**Comment by bwoodsend** [comment]
Not that I know of - we officially support glibc and musl - but I'm definitely game for adding others. Do you know of any docker base images or Linux distributions using ulibc? That would be easiest for us to work with. Otherwise it looks like the [prebuilt toolchain images on ulibc's website](https://www.uclibc.org/toolchains.html) don't look too awkward.

**Comment by rokm** [comment]
Does python support uclibc, though?
